### PR TITLE
chore(tools): switch to yarn

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Comment by ojongerius** [comment]
Addresses #65

File: README.md [hunk 1]
```diff
@@ -77,7 +83,7 @@ Deployment is normally done by CI.
 
 If you want to do a manual deployment:
 
-Configure your AWS credentials, see https://serverless.com/framework/docs/providers/aws/guide/credentials
+Configure your AWS credentials, see <https://serverless.com/framework/docs/providers/aws/guide/credentials>
```
Comment (ojongerius) [inline README.md:86]: Just curious what does change?

**Review by Bouncey — APPROVED** [review]

**Review by ojongerius — CHANGES_REQUESTED** [review]
Nice one @raisedadead.

I noticed @Bouncey raised a similar PR which included updating how to install serverless, here:

https://github.com/Bouncey/open-api/blob/feat/yarn/.travis.yml#L16

And   https://github.com/Bouncey/open-api/blame/feat/yarn/README.md#L63

See https://github.com/freeCodeCamp/open-api/pull/61/files

**Comment by raisedadead** [comment]
Hi @ojongerius 

Thanks for the feedback. I left both the Manual deployment and the CI part on purpose. 

Travis will automatically use Yarn or npm. So, its installation is not needed. Plus `npm i -g yarn` is a bad idea and should be avoided. This install form is deprecated.

For the manual deployment, it would make sense to use npm because its a global install for serverless and we are restricting yarn for our project. So others may or may not face issues?

My assumption and reason for the PR is that Yarn plays well for all scenarios and npm does not. Hence leaving global installs via npm will still mean Yarn is good to go and use if needed?

File: README.md (hunk 1 above)
Comment (raisedadead) [reply README.md:86]: This is a linting fix for avoiding bare URL. My editor said so. 🤷‍♂️

**Review by ojongerius — APPROVED** [review]

**Comment by ojongerius** [comment]
@raisedadead 🏆

**Comment by raisedadead** [comment]
Thanks @ojongerius !

**Pull request merged**